projects/packages/blocks/src/class-blocks.php

```diff
@@ -70,7 +70,7 @@ public static function jetpack_register_block( $slug, $args = array() ) {
 				$args['attributes'] = array_merge(
 					$args['attributes'],
 					array(
-						// Indicates that this block should display an upgrade nudge on the frontend when applicable.
+						// Indicates that this block should display an upgrade nudge on the frontend.
```

Comment:
It looks like this change needs its own Changelogger entry, because the change is in the blocks package. The other changelog entry just covers the plugin, but not this package, it seems. (This is the error from the GH action):

![image](https://user-images.githubusercontent.com/14988353/117743397-1e438e80-b24a-11eb-80e5-24ffb4d12cfe.png)